src/features/unified-share-modal/EmailForm.js

```diff
@@ -323,6 +327,17 @@ class EmailForm extends React.Component<Props, State> {
         return isValid;
     };
 
+    getContactPillVariant = (contactPill: SelectOptionProp): RoundPillVariant => {
```

Comment:
Happy to move this to a separate PR if that's preferred. This is the end goal of the change, but it's not directly related to the Pill Selector updates.

Reply:
i'd agree with moving this out to a separate pr.

Reply:
Sounds good, removed the USM updates